### ISSUE TITLE
Hello Dev

### ISSUE THREAD

**Issue by Anas-1554** [bryanbarrios/lil-docs-template#1]
Hello Dev, 

I just want to ask do you still work on this project?

**Comment by bryanbarrios** [comment]
Hey @Anas-1554!

I'm not working on it anymore, but maybe in the future I'll take it up again or create a new template with a better approach.

You could check the [shadcn-ui docs site](https://github.com/shadcn-ui/ui/tree/main/apps/www). It's great.

**Comment by Anas-1554** [comment]
Sure no worries, I will check It out. 

I just want to ask you have written this 
"No production ready, but you can explore the project." 

Can I work on this and use it in production. Cause I am not looking for something complicated.

**Comment by bryanbarrios** [comment]
Yes, you can. If you need something simple and don't require the missing features, it may work for you.

**Comment by Anas-1554** [comment]
Last thing, Can you tell what are the `features` you are talking about here?
&
Can you tell me what kind of `performance` you are talking about?
